src/mumble/Audio.cpp

```diff
@@ -25,13 +26,23 @@ LoopUser LoopUser::lpLoopy;
 CodecInit ciInit;
 
 void CodecInit::initialize() {
-	CELTCodec *codec = NULL;
+#ifdef USE_OPUS
+	OpusCodec *oCodec = new OpusCodec();
+	if (oCodec->isValid()) {
+		oCodec->report();
+		g.oCodec = oCodec;
+	} else {
+		delete oCodec;
```

Comment:
We should probably log/display some information if the opus codec should be used but is not available/found?